cuegui/cuegui/HostMonitor.py

```diff
@@ -122,6 +122,7 @@ def __filterAllocationSetup(self, layout):
 
         btn = QtWidgets.QPushButton("Filter Allocation")
         btn.setMaximumHeight(FILTER_HEIGHT)
+        btn.setStyleSheet(" width: 120px;");
```

Comment:
It would be great if we could find a way for this to be adaptive to the text contained within the button, so for example if the button's text grows the button will grow with it.

For example, could we set button's `padding` instead of setting an explicit size?

If so, can we set this in the stylesheet (`darkpalette.qss`) so it fixes this for ALL buttons, instead of just these two?

Reply:
Ya, that definetly sounds better.
Will do that.

Reply:
@bcipriano Updated the PR. Please review.